grunttasks/verify-styles.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+var stripSpecialChars = require('./lib/strip-special-chars');
+
+module.exports = function (grunt) {
+  'use strict';
+  grunt.registerMultiTask('verify-styles', 'Verify whether the translated styles are valid', function (){
+    var validStylesFile = path.join(this.files[0].dest, grunt.config.process('<%= yeoman.validStylesFile %>'));
+    var validStylesArray = grunt.file.readJSON(validStylesFile);
+    var filesToBeChecked = this.files[0].src;
+
+    var hasErrors = 0;
+    var generatedStylesRegex = /style=('|")[\S\s]*?('|")/ig;
```

Comment:
I forgot, in HTML5, the quotes are strictly optional. A style can be considered valid w/o the quotes.

```
style=backbroundImage:none
```

Instead of using regular expressions to check styles, we may want to use the full html parser in https://github.com/mozilla/fxa-content-server-l10n/pull/63/files#diff-89412030af742d9041e0c356ccf997c7R5, moving the checks from this file there.